DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_robustness_macs.py

```diff
@@ -0,0 +1,161 @@
+import pytest
+import asyncio
+
+from dent_os_testbed.lib.bridge.bridge_link import BridgeLink
+from dent_os_testbed.lib.ip.ip_link import IpLink
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_setup_streams,
+    tgen_utils_start_traffic,
+    tgen_utils_stop_traffic,
+    tgen_utils_get_loss
+)
+
+pytestmark = [
+    pytest.mark.suite_functional_bridging,
+    pytest.mark.asyncio,
+    pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")
+]
+
+async def test_bridging_robustness_macs(testbed):
+    """
+    Test Name: test_bridging_robustness_macs
+    Test Suite: suite_functional_bridging
+    Test Overview: Verify that robustness macs learning.
+    Test Author: Kostiantyn Stavruk
+    Test Procedure:
+    1.  Init bridge entity br0.
+    2.  Set ports swp1, swp2, swp3, swp4 master br0.
+    3.  Set entities swp1, swp2, swp3, swp4 UP state.
+    4.  Set bridge br0 admin state UP.
+    5.  Set ports swp1, swp2, swp3, swp4 learning ON.
+    6.  Send traffic to swp1 to learn source increment address
+        00:00:00:00:00:35 with step '00:00:00:00:10:00' and count 4000.
+    7.  Verify that address have been learned and removed from previous learned port.
```

Comment:
What is the previous port? There is no step where we resend to another port.
How this test differs to test_bridging_full_fdb_traffic?

BTW: You can move all those tests to the test_bridging_learning.py.
Please avoid creating a separate module for every test case.